help/destinations/catalog/personalization/pega.md

```diff
@@ -1,13 +1,17 @@
 ---
-title: Pega Customer Decision Hub connection
-description: Use the Pega Customer Decision Hub destination in Adobe Experience Platform to send profile attributes and audience membership data to Pega Customer Decision Hub for next-best-action decisioning.
+title: (V1) Pega CDH Realtime Audience connection
+description: Use the Pega Customer Decision Hub Realtime Audience destination in Adobe Experience Platform to send profile attributes and audience membership data to Pega Customer Decision Hub for next-best-action decisioning.
 exl-id: 0546da5d-d50d-43ec-bbc2-9468a7db4d90
 ---
-# Pega Customer Decision Hub connection
+# Pega CDH Realtime Audience connection
+
+>[!IMPORTANT]
+>
+>This version of the Pega Customer Decision Hub Realtime Audience destination supports only a single Pega Customer Decision application. If you have multiple Pega Customer Decision Hub applications configured, you need to use the (V2) Pega CDH Realtime Audience destination connector.
```

Comment:
Please add a link to the "(V2) Pega CDH Realtime Audience destination connector" documentation page.

Reply:
Adobe did not publish the V2 version yet. I have summitted the request. 

The pull request for the documentation is https://github.com/AdobeDocs/experience-platform.en/pull/438

Reply:
You can still add the link to the pega-v2.md file. Please add that too.

Reply:
Updated